packages/frolint/src/utils/isInstanceOfNodeError.ts

```diff
@@ -0,0 +1,11 @@
+/* globals NodeJS */
+
+/**
+ * This is necessary when we want to use information available on a Nodejs specific `ErrnoException` object.
+ * Since the runtime instance is one from just `Error`, this function helps us narrowing at the type level.
+ */
+export function isInstanceOfNodeError<T extends new (...args: any) => Error>(
+  value: unknown
+): value is InstanceType<T> & NodeJS.ErrnoException {
```

Comment:
`T` is not constrained by the arguments, and therefore not much useful nor accurate here.

```suggestion
export function isInstanceOfNodeError(value: unknown): NodeJS.ErrnoException {
```

Reply:
That is a good point, I didn't really look into that part after my copy/pasta 🙄